src/app/home/home.component.html

```diff
@@ -310,7 +310,8 @@
                             }"
                         >
                             <div class="information-circle-flex">
-                                <div>
+                                <div *ngIf="textMarking.subtype === 'gabim gramatikor'">
+                                    <i class="bi bi-stars"></i>
```

Comment:
Apply if ngif only to the icon, everything else will be the same, and change the comparison to `... === 'grammatical'`.

Reply:
On it